instructor/retry.py

```diff
@@ -31,8 +31,8 @@ def reask_messages(response: ChatCompletion, mode: Mode, exception: Exception):
     if mode == Mode.ANTHROPIC_TOOLS:
         # TODO: we need to include the original response
         yield {
-            "role": "assistant",
-            "content": f"Validation Errors found:\n{exception}\nRecall the function correctly, fix the errors",
+            "role": "user",
+            "content": f"Validation Errors found:\n{exception}\nReca ll the function correctly, fix the errors from\n{response.content}",
```

Comment:
Might be better to use the following format from official documentation to indicate an error?
```json
{
  "role": "user",
  "content": [
    {
      "type": "tool_result",
      "tool_use_id": "toolu_01A09q90qw90lq917835lq9",
      "content": "ConnectionError: the weather service API is not available (HTTP 500)",
      "is_error": true
    }
  ]
}
```

Reply:
oh this is great are you able to kick of a PR

Reply:
Ok I will take a look at it tomorrow, feel free to continue this PR though